cmd/flux/create_source_git.go

```diff
@@ -122,6 +122,7 @@ func init() {
 
 func newSourceGitFlags() sourceGitFlags {
 	return sourceGitFlags{
+		keyAlgorithm:  "rsa",
```

Comment:
Can't we use here the RSAPrivateKeyAlgorithm from the manifestgen package? this way we don't hardcode it twice